tools/Makefile.am.inc

```diff
@@ -30,3 +30,19 @@ eos_network_id_CFLAGS = \
 	$(NULL)
 
 eos_network_id_LDADD = $(EOS_INSTRUMENTATION_LIBS)
+
+bin_SCRIPTS = eos-label-location
+
+do_edit = sed \
+	-e "s|\@sysconfdir\@|$(sysconfdir)|"
+
+eos-label-location: tools/eos-label-location.in Makefile
+	$(AM_V_GEN) $(do_edit) $< >$@
```

Comment:
sed -i is atomic, but this was not, as relying on shell redirect due to different path - fixed.